apps/frontend/src/modules/meeting/slices/actions.ts

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+import {
+	type AsyncThunkConfig,
+	type MeetingCreateRequestDto,
+	type MeetingResponseDto,
+} from "~/libs/types/types.js";
+
+import { name as sliceName } from "./meeting.slice.js";
+
+const createMeeting = createAsyncThunk<
+	MeetingResponseDto,
+	{ data: MeetingCreateRequestDto },
+	AsyncThunkConfig
+>(
+	`${sliceName}/create-meeting`,
+	async ({ data }, { extra, rejectWithValue }) => {
+		const { meetingApi } = extra;
+
+		try {
+			const meeting = await meetingApi.create(data);
+
+			return meeting;
+		} catch (error: unknown) {
+			return rejectWithValue({
+				message: error instanceof Error ? error.message : String(error),
+			});
```

Comment:
Do we need this `rejectWithValue`? I think errorHandler middleware will handle the error even if you don't reject explicitly